datalad_next/processors/decode_processor.py

```diff
@@ -0,0 +1,75 @@
+""" Data processor that decodes bytes into strings """
+
+from __future__ import annotations
+
+from typing import (
+    Generator,
+    Iterable,
+)
+
+
+__all__ = ['decode_processor']
+
+
+def decode_processor(iterable: Iterable[bytes],
```

Comment:
What about `decode_processor` -> `decode_bytes`? It is made for this input only, and give an "active" name that is similar to other itertools.

Reply:
will do

Reply:
done in #538